src/subdomains/core/referral/reward/ref-reward-out.service.ts

```diff
@@ -19,15 +19,21 @@ export class RefRewardOutService {
     private readonly dexService: DexService,
   ) {}
 
-  async payoutTransactions(): Promise<void> {
+  async checkPaidTransaction(): Promise<void> {
     try {
       const transactionsPaidOut = await this.refRewardRepo.find({
         where: { status: RewardStatus.PAYING_OUT },
         relations: ['user'],
       });
 
       await this.checkCompletion(transactionsPaidOut);
+    } catch (e) {
+      console.error('Failed to check paid transactions', e);
```

Comment:
Minor detail: use a separator (colon) for logs to improve readability in the log:
```ts
console.error('Failed to check paid transactions:', e);
```
(same below)